src/components/App/App.css

```diff
@@ -0,0 +1,23 @@
+/* App.css */
+body {
```

Comment:
According to BEM, it's not allowed to use **tags** in `css` for styling. You need to add **class names** for the elements 


**Please, correct it everywhere in the code**